xarray/core/ops.py

```diff
@@ -475,7 +475,11 @@ def inject_all_ops_and_reduce_methods(cls, priority=50, array_only=True):
 def inject_bottleneck_rolling_methods(cls):
     # standard numpy reduce methods
     for name in NAN_REDUCE_METHODS:
-        f = getattr(np, name)
+        try:
+            f = getattr(np, 'nan' + name)
+        except AttributeError:
+            # some versions of numpy dont have all the nan-safe methods
+            f = getattr(np, name)
```

Comment:
Which versions of numpy are missing these methods? in general I would rather give errors than silently yield inconsistent results.